core/assets/bundles/bundle_nl.properties

```diff
@@ -194,68 +194,68 @@ save.delete.confirm = Weet je zeker dat je dit opgeslagen spel wilt verwijderen?
 save.delete = Verwijderen
 save.export = Export opgeslagen spel
 save.import.invalid = [accent]Dit opgeslagen spel is ongeldig!
-save.import.fail = [crimson]Failed to import save: [accent]{0}
-save.export.fail = [crimson]Failed to export save: [accent]{0}
-save.import = Import Save
-save.newslot = Save name:
+save.import.fail = [crimson]Importeren van het opgeslagen spel mislukt: [accent]{0}
+save.export.fail = [crimson]Exporteren van het opgeslagen spel mislukt: [accent]{0}
+save.import = Importeer opgeslagen spel
+save.newslot = Naam opgeslagen spel:
 save.rename = Hernoemen
 save.rename.text = Nieuwe naam:
 selectslot = Selecteer een opgeslagen spel.
 slot = [accent]Slot {0}
 editmessage = Bericht bewerken
-save.corrupted = [accent]Save file corrupted or invalid!\nIf you have just updated your game, this is probably a change in the save format and [scarlet]not[] a bug.
+save.corrupted = [accent]Bestand beschadigd of ongeldig!\nAls je je spel net hebt bijgewerkt, is dit waarschijnlijk een wijziging in het opslagformaat en [scarlet]geen[] bug.
 empty = <empty>
-on = On
-off = Off
-save.autosave = Autosave: {0}
+on = Aan
+off = Uit
+save.autosave = Automatisch opslaan: {0}
 save.map = Map: {0}
-save.wave = Wave {0}
-save.mode = Gamemode: {0}
+save.wave = Golf {0}
+save.mode = Spelmodus: {0}
 save.date = Last Saved: {0}
-save.playtime = Playtime: {0}
-warning = Warning.
-confirm = Confirm
-delete = Delete
-view.workshop = View In Workshop
-workshop.listing = Edit Workshop Listing
-ok = OK
+save.playtime = Speeltijd: {0}
+warning = Waarschuwing.
+confirm = Bevestigen
+delete = Verwijderen
+view.workshop = In Workshop bekijken
+workshop.listing = Workshop lijst bewerken
```

Comment:
Translation error:
The word "Listing" means something along the lines of an entry in the Workshop catalog, it does not mean anything related to an actual list. Suggestion: "Workshop-Publicatie"

Reply:
Thnx, I'll change that